.github/workflows/release.yml

```diff
@@ -35,6 +35,19 @@ jobs:
         if: ${{ success() }}
         uses : ffurrer2/extract-release-notes@v1
 
+      - name: Get version
+        id: get_version
+        if: ${{ success() }}
+        run: echo "version=${GITHUB_REF/refs\/tags\//}" >> $GITHUB_ENV
+
+      - name: Create zip for sdkman
```

Comment:
The zip-file is also used for uploading to brew. Please rename task to reflect this, smth like `Create zip for dependency managers`